ts-fold-parsers.el

```diff
@@ -171,6 +172,14 @@
              (ts-fold-range-line-comment node offset "#")
            (ts-fold-range-c-like-comment node offset))))))
 
+(defun ts-fold-parsers-ocaml ()
+  "Rule sets for OCaml."
+  '((comment             . ts-fold-range-ocaml-comment)
+    (module_definition   . ts-fold-range-ocaml-module-definition)
+    (type_definition     . ts-fold-range-ocaml-type-definition)
+    (value_definition    . ts-fold-range-ocaml-value-definition)
+    ))
```

Comment:
Let's try not to leave parens onto new lines. Thanks!

Reply:
Ah, my bad, I'm not yet used to the elisp guidelines 😆

Reply:
Thanks!